src/components/AdminPanelLayout/UsersPane.tsx

```diff
@@ -0,0 +1,42 @@
+import {Table} from 'react-bootstrap';
+import React, {useState} from 'react';
+import {User} from '../../types/User';
+import {getUserList} from '../../api';
+import {useMount} from '../../hooks/useMount';
+import {HorizontallyCenteredSpinner} from '../util/HorizonallyCenteredSpinner';
+import {Layout} from '../../pages/Layout/Layout';
+
+
+const UsersPane = () => {
+  const [users, setUsers] = useState();
+  const [loading, setLoading] = useState(true);
+
+  useMount(async () => {
+    setUsers(await getUserList());
+    setLoading(false);
+  });
+
+  return <Layout>{loading ? <HorizontallyCenteredSpinner/> : (
+    <Table striped={true} bordered={true} hover={true}>
+      <thead style={{backgroundColor: '#adb5bd'}}>
+      <tr>
+        <th>Name</th>
+        <th>Email</th>
+        <th>Role</th>
+      </tr>
+      </thead>
+      <tbody>
+      {users.map((u: User) => (
+        <tr key={u.id} style={{cursor: 'pointer'}}>
+          <td>{u.firstName} {u.lastName}</td>
+          <td>{u.email}</td>
+          <td>{u.role}</td>
```

Comment:
I assume we are adding the ability for admins to change roles next sprint?

Reply:
Yeah that's a separate card from the one I was working on, I figured best to leave it for next sprint